crates/cxx-qt-gen/src/generator/cpp/signal.rs

```diff
@@ -7,7 +7,7 @@ use crate::{
     generator::{
```

Comment:
need to add the same support for our generated properties so that `self.on_property_changed(|value| ..);` works.

Reply:
We don't need this instead you can use the `#[inherit]` on the signals to reach this which means we don't need to duplicate a load of code.

Eg

```rust
#[cxx_qt::qobject]
pub struct MyObject {
        #[qproperty]
        my_property: bool,
}
#[cxx_qt::qsignals(MyObject)]
pub enum Signals {
    #[inherit]
    MyPropertyChanged,
}
...
self.as_mut().on_my_property_changed(|qobject| println!("{}", qobject.my_property()), ConnectionType::AutoConnection);
```

Then depending what happens with #61 there might be an argument for the new value, otherwise you can just read it back (same as in C++)

Reply:
So then this just needs documenting in the book

Reply:
Added some notes to the book.

Reply:
Although maybe there should be an example of this somewhere....